mindmeld/components/nlp.py

```diff
@@ -510,9 +510,11 @@ def _load(self, incremental_timestamp=None):
         model_path, incremental_model_path = path.get_domain_model_paths(
             app_path=self._app_path, timestamp=incremental_timestamp
         )
+        incremental_model_path_exist = os.path.exists(
+            incremental_model_path) if incremental_model_path is not None else False
```

Comment:
Should this check be done inside `get_domain_model_paths()` instead? It seems it should provide `incremental_model_path` as `None` if there's nothing there.

Reply:
Actually get_domain_model_paths() is used to generate the path for both loading and dumping. So it'll be hard without adding another parameter to check if I should return None when we're loading vs before dumping before the file is created